app/components/app_consent_component.html.erb

```diff
@@ -9,7 +9,7 @@
 <% if patient_session.consents.present? %>
   <% consents_grouped_by_parent.each do |summary, consents| %>
     <%= render AppDetailsComponent.new(summary:, open: open_consents?) do %>
-      <%= render AppConsentDetailsComponent.new(consents:) %>
+      <%= render_grouped_consents(consents) %>
```

Comment:
Just had another thought about making this a bit more readable:

```suggestion
      <%= render AppConsentSummaryComponent.new(**grouped_consents) %>
```

Reply:
Yep, good idea, added in [b7a8b34](https://github.com/nhsuk/manage-childrens-vaccinations/pull/1206/commits/b7a8b348e2c1ecc34071fea5aa42f8b38b19eb67)